src/main/java/com/corndel/pixmate/exercises/Exercise3.java

```diff
@@ -11,7 +13,19 @@ public class Exercise3 {
    * @returns {int} The count of numbers larger than 10
    */
   public static int gridLimit(int[][] grid) {
-    // TODO
-    return 0;
+int count = 0;
+
+    int[] flattenedGrid = Arrays.stream(grid)
```

Comment:
Wow good use of streams, they are advanced but I like the application. 

You could/should be able to do the whole process with streams.

```java
Arrays.stream(grid)
            .flatMapToInt(Arrays::stream)
            .filter(num -> num > 10)
            .count();
```